platforms/platform_if/par/platform_if.sdc

```diff
@@ -14,4 +14,4 @@ set_false_path -from [get_keepers *|platform_shim_ccip|c.ccip_async_shim|async_s
 ##
 ## Reset path to local memory clock after clock crossing.
 ##
-set_false_path -to [get_keepers *|platform_shim_avalon_mem_if|c.local_mem_reset_pipe[0]]
+set_false_path -to [get_keepers *|platform_shim_avalon_mem_if|c.mm_async*.local_mem_reset_pipe[0]]
```

Comment:
What does this settings do? local_mem_reset_pipe value set to 0?

Reply:
See Quartus timing constraints: https://www.altera.com/en_US/pdfs/literature/ug/ug_tq_tutorial.pdf